modules/ROOT/pages/index.adoc

```diff
@@ -3,26 +3,62 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the Anypoint Platform Private Cloud Edition. The Anypoint Platform Private Cloud Edition provides the following application components available in the online version of Anypoint Platform:
-
-* Runtime Manager: enables you to track and manage your deployed Mule applications
-* API Manager: enables you to manage your registered APIs
-* Anypoint Exchange: enables your organization to share various assets that can be reused across integration projects.
-* Access Management
-* Design Center, excluding Flow Designer
-
-The following components are not included in the online version of Anypoint Platform:
```

Comment:
Where did this go?

Reply:
OK - added the info back in - wasn't sure how much detail was needed.